index.bs

```diff
@@ -1997,93 +2082,64 @@ The [=remote end steps=] with |command parameters| are:
-       <code>url</code> field set to the result of the [=URL serializer=] given
-       |navigate status|'s url.
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextReloadCommand = {
+        method: "browsingContext.reload",
+        params: BrowsingContextReloadParameters
+      }
 
-    1. Return [=success=] with data |body|, and then run the following steps [=in
-       parallel=]:
+      BrowsingContextReloadParameters = {
+        context: BrowsingContext,
+        ?ignoreCache: boolean,
+        ?wait: ReadinessState,
+      }
+      </pre>
+   </dd>
+</dl>
 
-       1. Run the [=WebDriver-BiDi navigation started=] steps given |context|
-          and |navigate status|
+<div algorithm="remote end steps for browsingContext.reload">
+The [=remote end steps=] with |command parameters| are:
 
-  1. Run the [=WebDriver-BiDi navigation started=] steps given |context|
-     and |navigate status|
+1. Let |context id| be the value of the <code>context</code> field of
+   |command parameters|.
 
-     Note: this event was previously suppressed to ensure that it would come
-     after the command response in the case that |wait condition| is
-     "<code>none</code>".
+1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+   with |context id|.
 
-     Issue: Replace this suppression mechanism with an event queue.
+1. Assert: |context| is not null.
 
-  1. If |wait condition| is "<code>interactive</code>", let |event name| be
-     "<code>domContentLoaded</code>", otherwise let |event name| be
-     "<code>load</code>".
+1. Let |ignore cache| be the the value of the <code>ignoreCache</code> field of |command
+   parameters| if present, or false otherwise.
 
-  1. Let (|event received|, |status|) be [=await=] given «|event name|,
-      "<code>download started</code>", "<code>navigation aborted</code>",
-      "<code>navigation failed</code>"» and |navigation id|.
+1. Let |wait condition| be the value of the <code>wait</code> field of |command
+   parameters| if present, or "<code>none</code>" otherwise.
 
-  1. If |event received| is "<code>navigation failed</code>"
-     return [=error=] with [=error code=] [=unknown error=].
+1. Let |document| be |context|'s [=active document=].
 
-     Issue: Are we surfacing enough information about what failed and why with
-     an error here? What error code do we want? Is there going to be a problem
-     where local ends parse the implementation-defined strings to figure out
-     what actually went wrong?
+1. Let |url| be |document|'s <a spec=DOM>URL</a>.
 
-  1. Let |body| be a [=map=] matching the
-     <code>BrowsingContextNavigateResult</code> production, with the
-     <code>navigation</code> field set to |status|'s id, and the
-     <code>url</code> field set to the result of the [=URL serializer=] given
-     |status|'s url.
+1. Let |request| be a new [=/request=] whose URL is |url|.
```

Comment:
I looked around to see if browsing contexts have URLs, but it seems like not. That's good, this seems right.